docs/source/explanation/index.rst

```diff
@@ -0,0 +1,26 @@
+Explanation
+===========
+
+This section consists of an algorithms explanation and describes additional features that are supported by OTX.
+``algorithms`` section includes a description of all supported algorithms:
```

Comment:
Nit: ``Algorithms`` might be better. Link might be even better. What do you think?